src/plugin-slots/CourseAuthoringUnitSidebarSlot/README.md

```diff
@@ -1,9 +1,11 @@
 # CourseAuthoringUnitSidebarSlot
 
-### Slot ID: `org.openedx.frontend.authoring.course_unit_sidebar.v1`
+### Slot ID: `org.openedx.frontend.authoring.course_unit_sidebar.v2`
 
 ### Slot ID Aliases
-* `course_authoring_unit_sidebar_slot`
+* `course_authoring_unit_sidebar_slot_v2`
```

Comment:
The v2 slot doesn't have any alises, so this documentation is incorrect. Since the v2 slot is new, it *shouldn't* have any aliases, so the only change needed is removing the "Slot ID Aliases" section from this README file.

Reply:
@brian-smith-tcril Ah! My bad. I wanted to add an alias. Removing this should be straightforward.